src/NzbDrone.Core/Profiles/ProfileFormatItem.cs

```diff
@@ -7,7 +7,6 @@ namespace NzbDrone.Core.Profiles
     public class ProfileFormatItem : IEmbeddedDocument
     {
         [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
```

Comment:
Need to get rid of the JsonIgnore here as well

Reply:
Good catch, I missed a file!